charts/sda-svc/templates/api-certificate.yaml

```
@@ -0,0 +1,39 @@
+{{- if .Values.global.tls.enabled }}
+{{- if or .Values.global.tls.clusterIssuer .Values.global.tls.issuer }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ template "sda.fullname" . }}-api-certs
+spec:
+  # Secret names are always required.
+  secretName: {{ template "sda.fullname" . }}-api-certs
+
+  duration: 2160h # 90d
+
+  # The use of the common name field has been deprecated since 2000 and is
+  # discouraged from being used.
```

but then why are we using it..?

---

It's not used actively now, but will be needed if we decide to move to TLS based authentication instead of username:password for Postgres, RabbitMQ and so.

---

I see, thanks for the explanation! How about updating the comment a bit, then, eg:

```suggestion
  # discouraged from being used. However, it is still needed for TLS based authentication  for Postgres and other services.
```